### PR TITLE
introduction to seeder addon

### PR DESCRIPTION
**Pull request by ibrokethecloud**
PR introduces documentation for the seeder addon being made available as part of v1.2.0

### CODE REVIEW

File: docs/advanced/seeder.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+A node once Shutdown, allows users to subsequently power it on.
+
+![](/img/v1.2/seeder/PowerOn.png)
+
+
+## Hardware event aggregation
+
+If `Event` collection is enabled in `Out-of-Band Access`, then `seeder` leverages `redfish` to query the underlying hardware for information about component failures and fan temperatures.
+
+This information is correlated with Harvester nodes, and made available as kubernetes events
```
Comment (vickyhella) [inline docs/advanced/seeder.md:46]:
```suggestion
This information is associated with Harvester nodes and can be used as Kubernetes events.
```

File: docs/advanced/seeder.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Once he addon is enabled, users can `Edit Config` for a specific host, to access `Out-Of-Band` definition page
+
+![](/img/v1.2/seeder/EditConfig.png)
+
+![](/img/v1.2/seeder/OutOfBandAccess.png)
+
+`seeder` leverages `ipmi` for performing management operations on the underlying node hardware.
+
+`redfish` support is needed for optional hardware discovery and event detection capability.
```
[the remark this answers is not in the document]
Comment (vickyhella) [reply docs/advanced/seeder.md:29]: Adding a hyperlink to `redfish` is better

**Review by vickyhella — APPROVED** [review]
LGTM!

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

File: docs/advanced/seeder.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+sidebar_label: Seeder
+title: "Seeder"
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/dev/advanced/seeder"/>
+</head>
+_Available as of v1.2.0_
+
+The `harvester-seeder` addon allows you to perform out-of-band operations on underlying nodes.
```
Comment (LucasSaintarbor) [inline docs/advanced/seeder.md:12]:
```suggestion
The `harvester-seeder` addon lets you perform out-of-band operations on underlying nodes. 
```

File: docs/advanced/seeder.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/dev/advanced/seeder"/>
+</head>
+_Available as of v1.2.0_
+
+The `harvester-seeder` addon allows you to perform out-of-band operations on underlying nodes. 
+
+For bare-metal nodes that support redfish-based access, this addon can also discover hardware and hardware events, and then associate the hardware with the corresponding Harvester nodes.
```
Comment (LucasSaintarbor) [inline docs/advanced/seeder.md:14]:
```suggestion
This addon can also discover hardware and hardware events for bare-metal nodes that support redfish-based access and then associate the hardware with the corresponding Harvester nodes.
```

File: docs/advanced/seeder.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+To get started, you need to enable the `harvester-seeder` addon from the **Addons** page.
```
Comment (LucasSaintarbor) [inline docs/advanced/seeder.md:16]:
```suggestion
You must enable the `harvester-seeder` addon from the **Addons** page to get started.
```

File: docs/advanced/seeder.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+![](/img/v1.2/vm-import-controller/EnableAddon.png)
+
+Once the addon is enabled, find the desired host and click **Edit Config**, and then click the `Out-Of-Band Access` tab.
```
Comment (LucasSaintarbor) [inline docs/advanced/seeder.md:20]:
```suggestion
Once the addon is enabled, find the desired host and select **Edit Config** and go to the **Out-Of-Band Access** tab.
```

File: docs/advanced/seeder.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Hardware discovery and event detection require `redfish` support.
+
+## Power operations
+
+Once you've defined the out-of-band config for a node, you can put the node into `Maintenance` mode, which allows you to shut down or reboot the node as needed.
+
+![](/img/v1.2/seeder/ShutdownReboot.png)
+
+If a node is shut down, you can also click **Power On** to power it on again:
```
Comment (LucasSaintarbor) [inline docs/advanced/seeder.md:36]:
```suggestion
If a node is shut down, you can also select **Power On** to power it on again:
```

**Review by LucasSaintarbor — APPROVED** [review]
Small NITs. LGTM

**Pull request merged**